server/commands/expand.go

```diff
@@ -194,21 +226,6 @@ func (query *ExpandQuery) resolveTupleToUserset(
 		}
 	}
 
-	tuplesetRewrite := relation.GetRewrite().GetUserset()
-	if tuplesetRewrite != nil && reflect.TypeOf(tuplesetRewrite) != reflect.TypeOf(&openfgapb.Userset_This{}) {
```

Comment:
This case is covered in the `validation/validation.go` (`validateTuplesetRestrictions`) method now. A tuple encountered with a tupleset relation that involves a rewrite according to the model will now be skipped.